src/sentry/utils/services.py

```diff
@@ -79,7 +79,7 @@ def __getattr__(self, name):
         # in our metrics decorator.
         if self._metrics_path and callable(attr) and name in self._base.__all__:
             return metrics.wraps(
-                self._metrics_path, instance=name, tags={"backend": self._backend}
+                self._metrics_path, instance=name, tags={"backend": self._backend, "name": name}
```

Comment:
Ignoring how bad of a name it is, doesn't `instance` (the method name) already get bound as a tag?

Reply:
Oof, don't know why I missed that. Closing this.